feature/bgp/policybase/otg_tests/import_export_multi_test/import_export_multi_test.go

```diff
@@ -313,7 +329,9 @@ func configureImportExportMultifacetMatchActionsBGPPolicy(t *testing.T, dut *ond
 		}
 	}
 	communitySetRefsAddCommunities.SetCommunityMember(cs3)
-	communitySetRefsAddCommunities.SetMatchSetOptions(matchInvert)
+	if deviations.BGPConditionsMatchCommunitySetUnsupported(dut) {
```

Comment:
See above, this doesn't appear to do what the intent of the deviation is as written.

Reply:
Thanks for the reference, now I get it!